uint8 ROBOT_BASE=0
uint8 PREVIOUS_TARGET=1
uint8 OBJECT=2
uint8 ref_type
geometry_msgs/PoseStamped ee_pose
float64[] joint_pose
float64[] velocities
Landmark ref_landmark
